reporter.js

```diff
@@ -155,21 +156,20 @@ Reporter.prototype = {
   },
 
   report: function (prefix, data) {
+    var test = {
+      launcher: prefix,
+      result: data
+    }
+
     if (data.pending) {
       this.skipped++
     } else if (data.passed) {
       this.pass++
-      this.passedTests.push({
-        launcher: prefix,
-        result: data
-      })
+      this.passedTests.push(test)
     } else {
       this.out.write(this.failedTests.length === 0 ? 'FAILED TESTS\n\n' : '\n')
-      this.failedTests.push({
-        launcher: prefix,
-        result: data
-      })
-      testWriter.call(this, data, true)
+      this.failedTests.push(test)
+      testWriter.call(this, test, true)
```

Comment:
👍 That's why we weren't getting proper reporting when there was that exception raised, the one that was ultimately caused by `ember-cli-code-coverage` but wasn't being reported.